AVFoundation/AVCaptureIrisStillImageOutput.h

```diff
@@ -0,0 +1,12 @@
+#import <AVFoundation/AVFoundation.h>
+#import <Celestial/FigCaptureStillImageSettings.h>
+
+API_DEPRECATED("Removed in iOS 10", ios(9.0, 10.0))
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
why do we want to ignore this diagnostic here?

Reply:
Line 9, with reference to `AVCaptureStillImageOutput` will error without this. This class has been deprecated.
https://developer.apple.com/documentation/avfoundation/avcapturestillimageoutput?language=objc

Reply:
missed this, thanks. sounds good